### PR TITLE
Manually update dependencies + fix vulnerability 

### LINKED ISSUE #5888: Update dependencies to remove vulnerabilities and allow Dependabot to work
**Describe the bug**
Github notified us that two dependencies of ESLint have vulnerabilites and need to be updated.

This also somehow stopped Dependabot from working, so our dependency updates have to be done manually.

image
https://github.com/bbc/simorgh/network/alert/package-lock.json/minimist/open

image
https://github.com/bbc/simorgh/network/alert/package-lock.json/minimist/open/update-errors/13208576

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves https://github.com/bbc/simorgh/issues/5888

**Overall change:** 
Manually update dependencies, because Dependabot is failing to run due to a vulnerability.

**Code changes:**
- Update `package.json` and `package-lock.json`
- Update snapshots

**Updated dependencies**

- "@bbc/psammead-brand": "5.1.15",
- "@bbc/psammead-grid": "1.1.11",
- "@bbc/psammead-media-player": "2.7.5",
- "@bbc/psammead-most-read": "2.0.3",
- "@bbc/psammead-navigation": "6.0.5",
- "@bbc/psammead-story-promo-list": "4.0.5",
- "core-js": "3.6.4",
- "helmet": "3.21.3",
- "helmet-csp": "2.9.5",
- "moment-timezone": "0.5.28",
- "ramda": "0.27.0",
- "react": "16.13.0",
- "react-dom": "16.13.0",
- "@babel/core": "7.8.7",
- "@babel/plugin-proposal-object-rest-spread": "7.8.3",
- "@babel/plugin-proposal-throw-expressions": "7.8.3",
- "@babel/plugin-transform-runtime": "7.8.3",
- "@babel/polyfill": "7.8.7",
- "@babel/preset-env": "7.8.7",
- "@babel/preset-react": "7.8.3",
- "@babel/runtime": "7.8.7",
- "@bbc/psammead-test-helpers": "3.1.3",
- "@storybook/addon-knobs": "5.3.17",
- "@storybook/react": "5.3.17",
- "@storybook/theming": "5.3.17",
- "@testing-library/react": "9.5.0",
- "babel-eslint": "10.1.0",
- "chrome-launcher": "0.13.0",
- "eslint-config-airbnb": "18.1.0",
- "eslint-plugin-cypress": "2.10.3",
- "eslint-plugin-json": "2.1.1",
- "eslint-plugin-react": "7.19.0",
- "eslint-plugin-react-hooks": "2.5.0",
- "jest": "25.1.0",
- "jest-fetch-mock": "3.0.3",
- "jsdom": "16.2.1",
- "lint-staged": "10.0.8",
- "mocha": "7.1.0",
- "stylelint": "13.2.1",
- "timemachine": "0.3.1",
- "wait-on": "4.0.1",
- "webpack": "4.42.0",
- "webpack-bundle-analyzer": "3.6.1",

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Review by DenisHdz — APPROVED** [review]
There are conflicts with the bundleSize, other than that LGTM

**Review by hotinglok — APPROVED** [review]
LGTM, just went through and checked the package.json changes with the given list.

File: scripts/bundleSizeConfig.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 702,
-  MAX_SIZE: 765,
+  MIN_SIZE: 724,
+  MAX_SIZE: 775,
```
Comment (sareh) [inline scripts/bundleSizeConfig.js]:
```suggestion
  MAX_SIZE: 778,
```

**Comment by PriyaKR** [comment]
Regression looks good to me.

**Comment by sareh** [comment]
@jblaut minimist@1.2.3 is needed. 
Screenshot 2020-03-17 at 15 46 48

**Pull request merged**